### DISCUSSION TITLE
RustDesk Server Support For FreeBSD and/or OPNsense?

### DISCUSSION THREAD

**Discussion by Majoraslayer** [rustdesk/rustdesk#4639] [General]
Any chance there's going to be development on support for FreeBSD or OPNsense to run a RustDesk server?  As long as my OPNsense router is up (and by extension my network), the RustDesk server should be up if this were possible.  OPNsense is based on FreeBSD, so if support for that operating system were to be added it would open the door to a useful and reliable way to keep a server up.

**Comment by tschettervictor** [comment]
A simple `pkg install rustdesk-server` will do the trick now that it is available.

**Reply by Majoraslayer** [reply]
I get a message stating that no matching package is available in the repositories.  Is it officially supported on FreeBSD now, and if so, do I need to somehow add a repository for it to my Opnsense install?

**Reply by DevJupyHUB** [reply]
Just follow this to enable the official FreeBSD repo on OPNSense:  https://medium.com/@mihakralj/the-direct-route-installing-freebsd-packages-on-opnsense-d002ac0c56b8

**Comment by tschettervictor** [comment]
I don’t use opnsense, but I know it is available via the official FreeBSD repo. You will probably have to add that somehow.
